requirements.txt

```diff
@@ -1,15 +1,58 @@
+alembic==1.0.10
+asn1crypto==0.24.0
+atomicwrites==1.3.0
+attrs==19.1.0
+autopep8==1.4.4
+certifi==2019.3.9
+cffi==1.12.3
+chardet==3.0.4
+Click==7.0
+coverage==4.5.3
+cryptography==2.6.1
+entrypoints==0.3
 flake8==3.7.5
 Flask==0.12.2
 Flask-Cors==3.0.4
 Flask-JSON==0.3.2
+Flask-Migrate==2.5.2
 Flask-Script==2.0.6
+Flask-SQLAlchemy==2.4.0
 google-api-python-client==1.6.7
+http-ece==1.1.0
+httplib2==0.12.3
+idna==2.8
+importlib-metadata==0.15
+itsdangerous==1.1.0
+Jinja2==2.10.1
+Mako==1.0.10
 MarkupSafe==1.1.0
+mccabe==0.6.1
+more-itertools==7.0.0
+ndg-httpsclient==0.5.1
 oauth2client==4.1.3
+pep8==1.7.1
+pluggy==0.12.0
+py==1.8.0
+py-vapid==1.5.0
+pyasn1==0.4.5
+pyasn1-modules==0.2.5
+pycodestyle==2.5.0
+pycparser==2.19
 pyfcm==1.4.5
-virtualenv==15.2.0
-pywebpush==1.8.0
+pyflakes==2.1.1
+pyOpenSSL==19.0.0
 pytest==4.3.0
-redis
-ndg-httpsclient
-pyopenssl
```

Comment:
Any reason why these packages are being removed?

Reply:
They were not removed also.